src/core/server/locales/sv/errors.ftl

```diff
@@ -0,0 +1,59 @@
+error-userNotFound = User ({$userID}) not found.
+error-notFound = Unrecognized request URL ({$method} {$path}).
+error-tokenInvalid = Invalid API Token provided.
+
+error-tokenNotFound = Specified token does not exist.
+error-emailAlreadySet = Email address has already been set.
+error-emailNotSet = Email address has not been set yet.
+error-duplicateUser =
+  Specified user already exists with a different login method.
+error-duplicateEmail = Specified email address is already in use.
+error-localProfileAlreadySet =
+  Specified account already has a password set.
+error-localProfileNotSet =
+  Specified account does not have a password set.
+error-usernameAlreadySet = Specified account already has their username set.
+error-usernameContainsInvalidCharacters =
+  Provided username contains invalid characters.
+error-usernameExceedsMaxLength =
+  Username exceeds maximum length of {$max} characters.
+error-usernameTooShort =
+  Username must have at least {$min} characters.
+error-passwordTooShort =
+  Password must have at least {$min} characters.
+error-emailInvalidFormat =
+  Provided email address does not appear to be a valid email.
+error-emailExceedsMaxLength =
+  Email address exceeds maximum length of {$max} characters.
+error-internalError = Internal Error
+error-tenantInstalledAlready = Tenant has already been installed already.
+error-userNotEntitled = You are not authorized to access that resource.
+error-storyNotFound = Story ({$storyID}) not found.
+error-commentNotFound = Comment ({$commentID}) not found.
+error-invalidCredentials = Email and/or password combination incorrect.
+error-toxicComment = Are you sure? The language in this comment might violate our community guidelines. You can edit the comment or submit it for moderator review.
+error-spamComment = The language in this comment looks like spam. You can edit the comment or submit it anyway for moderator review.
+error-userAlreadySuspended = The user already has an active suspension until {$until}.
+error-userAlreadyBanned = The user is already banned.
+error-userBanned = Your account is currently banned.
+error-userSuspended = Your account is currently suspended until {$until}.
+error-integrationDisabled = Specified integration is disabled.
+error-passwordResetTokenExpired = Password reset link expired.
+error-emailConfirmTokenExpired = Email confirmation link expired.
+error-rateLimitExceeded = You're trying to do that too many times. Please wait and try again.
+error-inviteTokenExpired = Invite link has expired.
+error-inviteRequiresEmailAddresses = Please add an email address to send invitations.
+error-passwordIncorrect = Incorrect password. Please try again.
+error-usernameAlreadyUpdated = You may only change your username once every { framework-timeago-time }.
+error-persistedQueryNotFound = The persisted query with ID { $id } was not found.
+error-rawQueryNotAuthorized = You are not authorized to execute this query.
```

Comment:
Hey @felixdro are you able to translate this as well? If you want to do it later, just please remove this file - it always defaults to English if there are missing translations. Thank you!

Reply:
This file is now partially translated too.

Reply:
This file is 100 % translated now